Testing/SystemTests/tests/framework/ReflectometryISISCalibrationPOLREFTest.py

```
@@ -0,0 +1,56 @@
+
+        for i in range(len(group_ws)):
+            spec_index = self._workspace_index_for_spectrum_number(group_ws[i], int(round(self._SPECULAR_PIXEL)))
+            self._check_geometry_changed_in_polref_scattering_plane(spec_index, group_ws[i], output_ws[i])
+
+    def validate(self):
+        return self._OUTPUT_FILE, self._REFERENCE_FILE
+
+    def requiredFiles(self):
+        return [self._POLREF_DATA_FILE + ".nxs", self._POLREF_CALIBRATION_MAP]
+
+    def _check_geometry_changed_in_polref_scattering_plane(self, spec_index, input, output):
```

Ruff (0.16.0)

[error] Function argument `input` is shadowing a Python builtin

(A002)